views/layout.pug

```diff
@@ -40,9 +40,12 @@ body
   include modal
   script(src="https://code.jquery.com/jquery-3.3.1.min.js")
   script(src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js")
+  script.
+    var fpjsRollbarClientId = !{JSON.stringify(fpjsRollbarClientId)};
```

Comment:
Minor: I think we could've used const here.

Reply:
It depends on what browsers the application must support. I'll switch to ES6 syntax if you want.